### DISCUSSION TITLE
Duplicate of a stat under SuperSmash section of player endpoint

### DISCUSSION THREAD

**Discussion by 0x26e** [HypixelDev/PublicAPI#385] [Ideas]
```python
response["player"]["stats"]["SuperSmash"]["smashLevel"]
response["player"]["stats"]["SuperSmash"]["smash_level_total"]
```
Both of these are in the same section, and represent the exact same stat, and are always equal to each other from what I've tested/observed.

We should probably remove one of these to prevent redundancy? Most other stats in this section use snake case, so it would probably be best to remove the camel case version.

**Comment by mdashlw** [comment]
There's no reason to remove this. They are probably both used, spending time on updating a legacy game and migrating all the players would achieve nothing.
